pkg/registry/controller/rest.go

```diff
@@ -47,6 +47,12 @@ func (rcStrategy) NamespaceScoped() bool {
 func (rcStrategy) PrepareForCreate(obj runtime.Object) {
 	controller := obj.(*api.ReplicationController)
 	controller.Status = api.ReplicationControllerStatus{}
+	if len(controller.Spec.Selector) == 0 {
```

Comment:
We should also update the descriptions in the v1betaN/types.go files to document the default values.